File: class23_homework/A-Ivaylo-Hristov-13/Ruby/data/MessageScreen.rb
class MessageScreen
	def newFlight
		welcomeScreen
		printMessage
		planeLogo
		printMessage
	end

	private
	def welcomeScreen
		@message = <<-STRING.gsub("\t", "")
			__          __  _
			\\ \\        / / | |
			 \\ \\  /\\  / /__| | ___ ___  _ __ ___   ___
			  \\ \\/  \\/ / _ \\ |/ __/ _ \\| '_ ` _ \\ / _ \\
			   \\  /\\  /  __/ | (_| (_) | | | | | |  __/
			    \\/  \\/ \\___|_|\\___\\___/|_| |_| |_|\\___|
			  _          _____                  _
			 | |        |_   _|           /\\   (_)
			 | |_ ___     | |_   _____   /  \\   _ _ __
			 | __/ _ \\    | \\ \\ / / _ \\ / /\\ \\ | | '__|
			 | || (_) |  _| |\\ V / (_) / ____ \\| | |    
			  \\__\\___/  |_____\\_/ \\___/_/    \\_\\_|_|
			
			
			***** Press enter to prepare a plane! *****
		STRING
	end
	
	def planeLogo
	@message = <<-STRING.gsub("\t", "")
	
		_____
		 \\ U \\__      _____
		  \\   \\/_______\\___\\_____________
		   < /_/   .....................  `-.
		   `-----------,----,--------------'
		             _/____/
		
		
		
		*** Plane is ready! All passengers get in line! ***
	STRING
	end
	
	def printMessage
		puts @message
		gets
	end
end

File: class23_homework/A-Ivaylo-Hristov-13/Java/src/homework23/MessageScreen.java
package homework23;

public class MessageScreen {

	public String message;

	public void newFlight() {
		welcomeScreen();
		printMessage();
		planeLogo();
		printMessage();
	}

	private void welcomeScreen() {
		message = "__          __  _" + "\n\\ \\        / / | |"
				+ "\n \\ \\  /\\  / /__| | ___ ___  _ __ ___   ___"
				+ "\n  \\ \\/  \\/ / _ \\ |/ __/ _ \\| '_ ` _ \\ / _ \\"
				+ "\n   \\  /\\  /  __/ | (_| (_) | | | | | |  __/"
				+ "\n    \\/  \\/ \\___|_|\\___\\___/|_| |_| |_|\\___|"
				+ "\n  _          _____                  _"
				+ "\n | |        |_   _|           /\\   (_)"
				+ "\n | |_ ___     | |_   _____   /  \\   _ _ __"
				+ "\n | __/ _ \\    | \\ \\ / / _ \\ / /\\ \\ | | '__|"
				+ "\n | || (_) |  _| |\\ V / (_) / ____ \\| | |    "
				+ "\n  \\__\\___/  |_____\\_/ \\___/_/    \\_\\_|_|"
				+ "\n\n\n***** Press enter to prepare a plane! *****";
	}

	private void planeLogo() {
		message = "\n   _____" + "\n   \\ U \\__      _____"
				+ "\n    \\   \\/_______\\___\\_____________"
				+ "\n     < /_/   .....................  `-."
				+ "\n     `-----------,----,--------------'"
				+ "\n               _/____/"
				+ "\n\n\n\n*** Plane is ready! All passengers get in line! ***";
	}

	public void printMessage() {
		System.out.println(message);
		System.console().readLine();
	}

}
